http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutes.scala

```diff
@@ -125,7 +125,7 @@ class ProxyRoutes(proxyService: ProxyService, corsSupport: CorsSupport)(
    * Extracts the user bearer token from an Authorization header.
    */
   private def extractTokenFromHeader: Directive1[Option[String]] =
-    optionalHeaderValueByType[`Authorization`](()).map(headerOpt => headerOpt.map(h => h.credentials.token))
```

Comment:
I wonder why `Authorization` was in quotes before.

Reply:
I think `(backtick)Authorization(backtick)` was the literal class name before. I'm going to have to get used to `foo(Type)` instead of `foo[Type]`

Reply:
ugh indeed...`foo(Type)` is so confusing 😕